src/pages/SettingsPage.js

```diff
@@ -28,14 +26,16 @@ import { ConfirmIcon } from '../widgets/icons';
 import { DataTablePageView, PageInfo } from '../widgets';
 import { DataTablePageModal } from '../widgets/modals';
 
-import globalStyles from '../globalStyles';
 import { generalStrings, buttonStrings } from '../localization';
 import { selectCurrentUserPasswordHash } from '../selectors/user';
 import { PermissionActions } from '../actions/PermissionActions';
-import { createRecord } from '../database/utilities/index';
-import BreachManager from '../bluetooth/BreachManager';
-import { VaccineDataAccess } from '../bluetooth/VaccineDataAccess';
 import { useLoadingIndicator } from '../hooks/useLoadingIndicator';
+import { createVaccineData } from '../database/utilities/dataGenerators';
+import { Slider } from '../widgets/Slider';
+import { FlexRow } from '../widgets/FlexRow';
+import globalStyles, { APP_FONT_FAMILY, DARK_GREY, SUSSOL_ORANGE } from '../globalStyles';
```

Comment:
Suggestion was to import from `../globalStyles` and not `/colors` as the FONT wasn't getting picked up.